scenarioo-server/src/main/java/org/scenarioo/rest/application/DocumentationPathLogic.java

```diff
@@ -0,0 +1,64 @@
+/* scenarioo-server
+ * Copyright (C) 2014, scenarioo.org Development Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.scenarioo.rest.application;
+
+import java.io.File;
+
+import javax.servlet.ServletContextEvent;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+
+/**
+ * Provides methods to get the configured path to the documentation and the config file name.
+ */
+class DocumentationPathLogic {
```

Comment:
@mi-we Could we rename this to `ScenariooDataPathLogic`  - finde den Namen "ScenariooData" besser als "Documentation" - sind ja alle Scenarioo Daten (inkl. config) und die Env Variable heisst auch so.

Reply:
I did it :-) - see other branch by me!